apps/meteor/ee/app/livechat-enterprise/server/lib/logger.ts

```diff
@@ -7,3 +7,4 @@ export const queueLogger = logger.section('Queue');
 export const helperLogger = logger.section('Helper');
 export const cbLogger = logger.section('Callbacks');
 export const bhLogger = logger.section('Business-Hours');
+export const ohLogger = logger.section('On-Hold');
```

Comment:
:kek:

Reply:
Lol! I'd defined it before so as to use it within this new service. However ended up defining a new logger over there and not using it. Glad you caught it. Thanks!